src/main/service/listManipulation/CrimeListsService.ts

```diff
@@ -1,8 +1,8 @@
 import moment from 'moment-timezone';
```

Comment:
Somewhere in this case, there is a method called "findUnallocatedCasesInCrownDailyListData" which is used by other lists too. This doesnt sound very generic so consider renaming it

Reply:
This seems to be used by CrownDailyListController only. If so could just move it to the daily list controller class. Strange that I can see some tests for other lists using that too. Need to find out more

Reply:
Gonna leave this as is for this round of refactoring - I don't feel all that comfortable with funcs in controller that aren't request types.